extras/apigee/oauth-config.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START gke_boa_kubernetes_manifests_config_configmap_oauth_config]
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: oauth-config
+data:
+  DEMO_OAUTH_CLIENT_ID: "{APIGEE_CLIENT_ID}"
+  DEMO_OAUTH_REDIRECT_URI: "https://{APIGEE_DOMAIN}/oauth/callback"
```

Comment:
Are these substituted in manually? If so, this step should be in the README at minimum

Reply:
This is explained in line 11 of the README